unison-cli/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -292,33 +333,30 @@ run version dir configFile stanzas codebase = do
       -- output ``` and new lines then call transcriptFailure
       dieWithMsg :: forall a. String -> IO a
       dieWithMsg msg = do
-        executable <- getProgName
         output "\n```\n\n"
         appendFailingStanza
         transcriptFailure out $ Text.unlines [
           "\128721", "",
           "The transcript failed due to an error in the stanza above. The error is:", "",
-          Text.pack msg, "",
-          "Run `" <> Text.pack executable <> " --codebase " <> Text.pack dir <> "` " <> "to do more work with it."]
```

Comment:
This invocation moved up a layer to `unison-cli/Main.hs` so we didn't need to manually strip it out of the output in `Transcripts.hs`